src/ppx/ppx_lwt.ml

```diff
@@ -407,11 +407,11 @@ let mapper =
 
 let args =
   Arg.([
-    "-no-debug", Clear debug, "disable debug mode";
-    "-log", Set log, "enable logging";
-    "-no-log", Clear log, "disable logging";
-    "-no-sequence", Clear sequence, "disable sequence operator";
-    "-no-strict-sequence", Clear strict_seq, "allow non-unit sequence operations";
+    "-no-debug", Clear debug, " disable debug mode";
+    "-log", Set log, " enable logging";
+    "-no-log", Clear log, " disable logging";
+    "-no-sequence", Clear sequence, " disable sequence operator";
+    "-no-strict-sequence", Clear strict_seq, " allow non-unit sequence operations";
```

Comment:
Thanks for doing this. I just tried looking at the PPX output, ouch.

Reply:
Help* output.

Reply:
@kandu, I cherry-picked this into Lwt `master`, see 35a727014521f93d2044a4fc4e0e7098158840b4. Thanks again!